pywhy_stats/discrepancy/bregman.py

```diff
@@ -0,0 +1,205 @@
+"""Bregman (conditional) discrepancy test.
+
+Also known as a conditional k-sample test, where the null hypothesis is that the
+conditional distributions are equal across different population groups. The
+Bregman tests for conditional divergence using correntropy.
+
+Returns
+-------
+PValueResult
+    The result of the test, which includes the test statistic and pvalue.
+"""
+
+from typing import Callable, Optional, Tuple
+
+import numpy as np
+from numpy.typing import ArrayLike
+from sklearn.base import BaseEstimator
+from sklearn.preprocessing import LabelBinarizer
+
+from pywhy_stats.kernel_utils import _preprocess_kernel_data, corrent_matrix, von_neumann_divergence
+
+from ..pvalue_result import PValueResult
+from .base import _compute_propensity_scores, _preprocess_propensity_data, compute_null
+
+
+def condind(
+    X: ArrayLike,
+    Y: ArrayLike,
+    group_ind: ArrayLike,
+    kernel: Optional[Callable[[ArrayLike], ArrayLike]] = None,
+    null_sample_size: int = 1000,
+    normalize_data: bool = False,
+    propensity_model=None,
+    propensity_weights=None,
```

Comment:
Something is off with the formatting here.

Reply:
Fixed.